sample-queries/sample-queries.json

```diff
@@ -1845,6 +1845,204 @@
       "tip": "This query requires the TI indicator external id. To find the ID, you can run: GET https://graph.microsoft.com/beta/security/tiIndicators?$top=5",
       "skipTest": false
     },
+    {
+      "id": "d4eb9650-719c-4c42-a124-c26549b671e7",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get articles",
+      "requestUrl": "/beta/security/threatIntelligence/articles",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "df9e38a2-b39e-42c8-9679-461a2f10fdef",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get an article",
+      "requestUrl": "/beta/security/threatIntelligence/articles/{articleId}",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "97cb3895-df00-4b7d-bf25-2b3d75fc8da0",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get indicators for an article",
+      "requestUrl": "/beta/security/threatIntelligence/articles/{articleId}/indicators",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "e98b1987-8104-4cca-8954-99038e6a25bf",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get an article indicator",
```

Comment:
Looking at the rest of the docs in your view, it looks like there are a lot of GETs that are just "an article Indicator". Since the UI adds the `GET` into the context, it kinda looks like we're double-explaining starting with "get".

![image](https://user-images.githubusercontent.com/126003046/227805032-9f20f320-c98c-417c-9988-a52bb53df8ae.png)

Reply:
<img width="404" alt="Screenshot 2023-03-27 at 10 50 12 AM" src="https://user-images.githubusercontent.com/126006033/228024796-6315017a-0418-4767-8061-81190071fa42.png">
Looks like they do both @joerattazzi-microsoft